[WIP] Asset Refactor Docs (#590)
frappe/erpnext_com: erpnext_com/www/docs/user/manual/en/asset/asset-movement.md

## 2. How to create an Asset Movement
1. Go to the Asset Movement list, click on New.

> 1. Select the Purpose. Mandatory fields will be changed based on purpose.

Reviewer: ```suggestion
1. Select the Purpose from 'Issue', 'Receipt', or 'Transfer'. Mandatory fields will be changed based on purpose.
```

1. Select a date.
1. Select Assets you want to move. Currenct Location / Custodian will be automatically fetched.

> 1. Select Reference Document Type (Optional)

Reviewer: ```suggestion
1. Select Reference Document Type (Optional).
```

> 1. Select Reference Document Name (Optional)

Reviewer: ```suggestion
1. Select Reference Document Name (Optional).
```

1. Save.
1. Submit.

> To make asset movement of a number of assets it is adviced to go to Asset List, select assets to be moved and click on **Make Asset Movement** from Actions menu on the top left. 

Reviewer: ```suggestion
To make an Asset Movement of a number of assets, it is advised to go to Asset List, select assets to be moved and click on **Make Asset Movement** from Actions menu on the top left. 
```
